packages/expo-ui/src/jetpack-compose/Loading/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { requireNativeView } from 'expo';
+import { ColorValue } from 'react-native';
+
+import { ExpoModifier } from '../../types';
+
+export type LoadingIndicatorVariant = 'default' | 'contained';
+
+export type LoadingIndicatorProps = {
+  /**
+   * The variant of the loading indicator.
+   * - `default`: A standard loading indicator with morphing shapes.
+   * - `contained`: A loading indicator inside a circular colored background.
+   *
+   * @default 'default'
+   * @platform android
+   */
+  variant?: LoadingIndicatorVariant;
+  /**
+   * The progress value of the indicator.
+   * - If provided: Determinate mode - morphs shapes based on progress value.
+   * - If null/undefined: Indeterminate mode - continuous morphing animation.
```

Comment:
```suggestion
   * - If provided: determinate mode, which morphs shapes according to the progress value.
   * - If `null` or `undefined`: Indeterminate mode, which uses continuous morphing animation.
```

I'd suggest using inline syntax highlights in the above, and avoid using en dashes as per Expo's writing style guide.

Reply:
Thanks for the review! I've applied your suggestion.